lib/video/screenRecording/firefox/firefoxWindowRecorder.js

```diff
@@ -146,7 +162,16 @@ module.exports = class FirefoxWindowRecorder {
         // Nothing to see here
       }
     }
-    await generateVideo(destination, this.baseDir);
+
+    let recordingDir = findRecordingDirectory(this.baseDir);
+    this.recordingStartTime = findRecordingStartTime(recordingDir);
+    this.timeToFirstFrame = findTimeToFirstFrame(recordingDir);
```

Comment:
Am I correct that you actually have two different directory walks now, one to find the outer directory, the other to enumerate inner frames in the recording directory?